docs/modelview/modelview.rst

```diff
@@ -324,6 +324,52 @@ Another detail specific to ``PathView`` worth noticing is the usage of the attac
 
 When transforming images or other complex elements on in ``PathView``, a performance optimization trick that is common to use is to bind the ``smooth`` property of the ``Image`` element to the attached property ``PathView.view.moving``. This means that the images are less pretty while moving but smoothly transformed when stationary. There is no point spending processing power on smooth scaling when the view is in motion, as the user will not be able to see this anyway.
 
+Table Models
+------------
+
+All views discussed until now present an array of items one way or another. Even the ``GridView`` expects the model to provide a one dimensional list of items. For two dimensional tables of data you need to use the ``TableView`` element.
+
+Conceptually, the ``TableView`` works just as all other views. It combines a ``model`` with a ``delegate`` to form a grid. If given a list type model it displays a single column, making it very similar to the ``ListView`` element. However, for table style models, it becomes more powerful. 
+
+In the example below, we setup a simple ``TableView`` with a custom model exposed from C++. At the moment, it is not possible to create table style models directly from QML, but in the 'Qt and C++' chapter the concept is explained. The running example is shown in the figure below.
```

Comment:
set up
(setup is a noun, not a verb)

Reply:
also this phrase "table style models" is unclear... someone might think it has something to do with visual styling.
"table-style models" or "table-oriented models" perhaps

Reply:
Good points (and thanks for finally clarifying setup vs set up :-) ).